packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -155,8 +156,12 @@ export class TeamsBotImpl {
       group_name
     );
 
-    await handler?.next(ProgressBarConstants.SCAFFOLD_STEP_UNZIP);
-    zipContent.extractAllTo(this.config.scaffold.workingDir!, true);
+    try {
+      await handler?.next(ProgressBarConstants.SCAFFOLD_STEP_UNZIP);
+      zipContent.extractAllTo(this.config.scaffold.workingDir!, true);
+    } catch (err) {
+      throw new InvalidPathError(this.config.scaffold.workingDir!, err);
```

Comment:
We can't say that all errors are caused by invalid path, maybe just ExtractTemplateError is enough. And we can suggest user to check whether path valid.

Reply:
OK